### ISSUE TITLE
Social club error 17 when mod is installed

### ISSUE THREAD

**Issue by Ryuzaki03** [LukeRoss00/gta5-real-mod#442]
CPU: AMD Ryzen 7 1800X Eight-Core Processor 3.60 GHz
GPU: NVIDIA GeForce GTX 1070 Ti
VR headset model: Oculus Quest 2
game version and build: latest release
release of the mod: 7
did you follow all the [Quick setup](https://github.com/LukeRoss00/gta5-real-mod#quick-setup) steps carefully, in particular step 4? Yes
which settings did you choose (H/M/L) when running the batch file RealConfig.bat? M
did the batch file report any errors? No
did you change the graphics settings manually? No
are you using any other mods? No, but I am intending to.

I never had this problem before when I was playing on vanilla, but when I turn on the game after installing this mod, the game would load for a few minutes in the title screen saying resetting social club, and then display a screen saying "Social Club failed to initialize. Error code: 17"
[ScriptHookV.log](https://github.com/LukeRoss00/gta5-real-mod/files/8984475/ScriptHookV.log)
[asiloader.log](https://github.com/LukeRoss00/gta5-real-mod/files/8984477/asiloader.log)
It says that I can't upload my RealVR.ini file because Github doesn't support it. How should I upload it?

**Comment by LukeRoss00** [comment]
Thanks for providing the logs and everything! In fact your logs indicate a problem: the final `CORE` section of `ScriptHookV.log` is completely missing. It should be something like this:

```
[18:54:05] INIT: DX init succeeded
[18:54:53] CORE: Requesting thread creation
[18:54:53] CORE: Creating threads
[18:54:54] CORE: Started control thread, id 37 active 1
[18:54:54] CORE: Started thread 'NativeTrainer.asi' (0x00007FFF44A1A3C0), id 38 active 1
[18:54:54] CORE: Started thread 'RealVR.asi' (0x00007FFF3316A580), id 39 active 1
[18:54:54] CORE: Launching main() for 'NativeTrainer.asi' (0x00007FFF44A1A3C0), id 38
[18:54:54] CORE: Launching main() for 'RealVR.asi' (0x00007FFF3316A580), id 39

```
instead yours ends at the "DX init succeeded" line, which means that my mod is not even getting loaded.
I'd suggest the following: rename `d3d11.dll` to `d3d11_ori.dll` and try launching the game, to see if it runs normally. If it's still broken, try verifying the game files with Steam. If even that is not enough, write again and we'll take it from there.

Also, I saw that you have a typo in your game installation path: it's "D:\steam lidrary" instead of "D:\steam library". That probably doesn't mean anything, but I was wondering if perhaps the Social Club launcher is getting confused by it.

**Comment by Ryuzaki03** [comment]
The social club problem is gone, but now the game isn"t launching in vr mode. It's just on desktop with the sides cut off. Also, changing the typo caused steam to make a new steam lidrary folder.

**Comment by Ryuzaki03** [comment]
also, it still ends with DX init succeeded.
[ScriptHookV.log](https://github.com/LukeRoss00/gta5-real-mod/files/8986185/ScriptHookV.log)
[asiloader.log](https://github.com/LukeRoss00/gta5-real-mod/files/8986188/asiloader.log)
[RealVR.ini.txt](https://github.com/LukeRoss00/gta5-real-mod/files/8986191/RealVR.ini.txt)

**Comment by LukeRoss00** [comment]
Yes, the typo was probably made when you selected a new library location in Steam. Did you have Steam verify the game files?

**Comment by Ryuzaki03** [comment]
yes, did.

**Comment by LukeRoss00** [comment]
You're not trying to play online, right? The mod only works for offline play.

If you can enter Story Mode correctly now and the launcher problem is gone, rename `d3d11_ori.dll` back to `d3d11.dll` and try the mod again.

**Comment by Ryuzaki03** [comment]
Yes, I am using Story mode only.

I changed the file name back and I'm back to the original state. I'm in vr mode, but the social club problem is back.

**Comment by LukeRoss00** [comment]
Let's check if your directory structure is correct. Open a command prompt, and enter the following commands:

```
cd /D D:\steam lidrary\steamapps\common\Grand Theft Auto V
dir /s > dir.txt
```

Then attach the resulting file `dir.txt` which will appear in the game folder.

**Comment by Ryuzaki03** [comment]
I am using Korean on my computer so some words are in Korean, but I think the important stuff is recognizable.
[dir.txt](https://github.com/LukeRoss00/gta5-real-mod/files/8988835/dir.txt)

**Comment by LukeRoss00** [comment]
Everything appears to be in order in your game directory.

Have you tried the official solution already? https://support.rockstargames.com/articles/115010233048/Social-Club-Error-Code-17

**Comment by Ryuzaki03** [comment]
Rename the settings.xml file to “settings.old" should I rename the new one created by RealConfig.bat?

**Comment by Ryuzaki03** [comment]
and I did try this several times already. didn"t work.

**Comment by Ryuzaki03** [comment]
it still doesn't work. still has the social club problem.

**Comment by LukeRoss00** [comment]
Did you also delete/rename `cfg.dat` and `pc_settings.bin` and added the game as an exception to your AV as recommended by Rockstar? If you already did, then I'm afraid I'm out of ideas...

**Comment by Ryuzaki03** [comment]
Renaming d3dll.dll kind of worked before. And according to Rockstar, social club error 17 has something to do with settings files. Do you know what d3dll.dll does to the game's settings?

**Comment by Ryuzaki03** [comment]
new problem. I deleted all my antivirus and changed d3dll into d3dll_ori and now when I launch the game, rockstar games launcher login screen pops up, but the login button is in some loading mode and isn't letting me log in.
![화면 캡처 2022-06-28 203049](https://user-images.githubusercontent.com/108175572/176168254-5d793d52-a967-4c89-9558-571b678bc6ec.png)

**Comment by Ryuzaki03** [comment]
And when I change the file name back to normal, social club error 17 happens again.

**Comment by Ryuzaki03** [comment]
Maybe it's something to do with social club directly?

**Comment by Ryuzaki03** [comment]
I think I will be giving up on vr for now. Thank you for your time and good luck with your work.

**Comment by LukeRoss00** [comment]
I'm very sorry that I was unable to help you :-( What I can tell you is that `d3d11.dll` doesn't touch the game settings at all, so it must be some other cause. What AV program are you using? Were you able to create an exception for files in the game folder and the Social Club folder? I'm not sure what you mean by "deleted all my antivirus"
